### PR TITLE
API change: values vs defaultValues

### PR DESCRIPTION
**Pull request by josepot**
Addresses #4

### CODE REVIEW

File: src/Slider/Slider.js [hunk 1]
```diff
@@ -37,15 +35,15 @@ class Slider extends Component {
   }
 
   componentWillMount() {
-    const { domain, defaultValues, step, reversed } = this.props
+    const { defaultValues, domain, step, reversed, values } = this.props
 
     this.updateRange(domain, step, reversed)
-    this.setState(this.getValues(defaultValues, reversed))
+    this.updateValues(values || defaultValues, reversed)
```
Comment (josepot) [inline src/Slider/Slider.js]: This is just so that it's backwards compatible.

File: src/Slider/Slider.js [hunk 2]
```diff
@@ -313,9 +313,13 @@ Slider.propTypes = {
...
   /**
    * An array of numbers. You can supply one for a value slider, two for a range slider or more to create n-handled sliders.
-   * The default values should correspond to valid step values in the domain.
+   * The values should correspond to valid step values in the domain.
    * The numbers will be forced into the domain if they are two small or large.
    */
+  values: PropTypes.array,
+  /**
+   * [DEPRECATED]: use `values` instead, which will also allow you to update the values from the outside.
+   */
```
Comment (josepot) [inline src/Slider/Slider.js]: The idea would be to stop supporting `defaultValues` in the next major version. It's just a suggestion, of course.

**Comment by sghall** [comment]
Alrighty then. Going to get into this. Like what you've done here.

**Comment by josepot** [comment]
Awesome! Thanks a lot @sghall ! Also, whenever you have a moment: could you please publish a new version with the latest changes? Thanks in advance!

**Comment by sghall** [comment]
Ok.  Just did a release (actually two).  Took care of a few other pending items and the lib is now 5kb instead of 16kb (!)  Would be really great if you can check it out and let me know what you think.  Like to get this library out there a little wider soon ;)  Cheers.

**Pull request merged**